pkg/handler/collector/gcs/gcs_test.go

```diff
@@ -77,16 +77,6 @@ func TestGCS_RetrieveArtifacts(t *testing.T) {
 		want:     doc,
 		wantErr:  false,
 		wantDone: true,
-	}, {
-		name: "last download time the same",
```

Comment:
this test case passes before the changes. Just need to add in the check again that the payload has a length greater than zero.

Reply:
Thanks, somehow I didn't realize that these weren't failing before the changes. I have updated the code!